kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
When we create containers for `gen` repo, they would checkout either a released version or specific commit of swagger-codegen and build using that. There, we can modify this file to record the commit it used to generate the code. It is a design I had in mind but if you have a better suggestion, let me know.